lib/plugins/unthreaded-signing/plugin.c

```diff
@@ -85,12 +85,12 @@ unthreaded_openssl_sign_hash(sv_unthreaded_plugin_t *self, const uint8_t *hash,
   // Borrow the |hash| by passing the pointer to |sign_data| for signing.
   self->sign_data.hash = (uint8_t *)hash;
   self->sign_data.hash_size = hash_size;
-
-  status = openssl_sign_hash(&self->sign_data);
-  if (status != SV_OK) return status;
   int idx = self->out_buffer_idx;
```

Comment:
What will now happen if `idx = MAX_BUFFER_LENGTH`? What kind of value is returned and is that what you want to return?

Reply:
This case it will return to SV_UNKNOWN_FAILURE. 
Since its a  unsupported behavior it should return SV_NOT_SUPPORTED.